test/tools/unified-spec-runner/match.ts

```diff
@@ -166,6 +166,16 @@ export function resultCheck(
       expect(actual[key]).to.have.all.keys(expectedSortKey);
       const objFromActual = { [expectedSortKey]: actual[key].get(expectedSortKey) };
       resultCheck(objFromActual, value, entities, path, checkExtraKeys);
+    } else if (key === 'createIndexes') {
+      for (const index of actual.indexes) {
+        expect(index).to.have.property('key').that.is.instanceOf(Map);
+        expect(
```

Comment:
I think we should be checking the size of the expected key and not just the actual one (similar to the above), since it's a limitation of the test framework (obviously the current code will still fail if the expected has more keys than the actual, but this way we're making it clear that it's a test input expectation incompatibility and not necessarily something wrong with the *result* of the test)

Reply:
Yea good call